src/cli.ts

```diff
@@ -108,6 +108,15 @@ export const main = defineCommand({
       description: "Open a tunnel using https://github.com/unjs/untun",
       required: false,
     },
+    ipc: {
+      type: "boolean",
+      description: "Listen on a Unix Domain Socket/Windows Pipe",
+      default: false,
+    },
+    ipcSocketName: {
+      type: "string",
+      description: "Name of socket/pipe",
```

Comment:
Maybe we can keep CLI simple with `--ipc [name]`?

Reply:
Yeah, that's true. This will be changed with the next push